pkg/config/config.go

```diff
@@ -314,29 +320,29 @@ type Put struct {
 
 // Project includes all project configuration
 type Project struct {
-	ProjectName   string      `yaml:"project_name,omitempty"`
-	Env           []string    `yaml:",omitempty"`
-	Release       Release     `yaml:",omitempty"`
-	Brew          Homebrew    `yaml:",omitempty"`
-	Scoop         Scoop       `yaml:",omitempty"`
-	Builds        []Build     `yaml:",omitempty"`
-	Archive       Archive     `yaml:",omitempty"` // TODO: remove this
-	Archives      []Archive   `yaml:",omitempty"`
-	NFPM          NFPM        `yaml:",omitempty"` // TODO: remove this
-	NFPMs         []NFPM      `yaml:"nfpms,omitempty"`
+	ProjectName   string    `yaml:"project_name,omitempty"`
+	Env           []string  `yaml:",omitempty"`
+	Release       Release   `yaml:",omitempty"`
+	Brew          Homebrew  `yaml:",omitempty"`
+	Scoop         Scoop     `yaml:",omitempty"`
+	Builds        []Build   `yaml:",omitempty"`
+	Archive       Archive   `yaml:",omitempty"` // TODO: remove this
+	Archives      []Archive `yaml:",omitempty"`
+	NFPM          NFPM      `yaml:",omitempty"` // TODO: remove this
+	NFPMs         []NFPM    `yaml:"nfpms,omitempty"`
 	Snapcraft     Snapcraft   `yaml:",omitempty"` // TODO: remove this
-	Snapcrafts    []Snapcraft `yaml:",omitempty"`
```

Comment:
this line should not have been deleted...

Reply:
done